brian2/stateupdaters/base.py

```diff
@@ -127,10 +138,16 @@ def apply_stateupdater(equations, variables, method):
             if the_method is None:
                 raise ValueError(('No stateupdater that is suitable for the '
                                   'given equations has been found.'))
-            logger.info('Using state updater: %r' % the_method)
+            if group_name is not None:
+                msg_text = ("No numerical integration method specified for group "
+                            "'{group_name}', choosing method '{method}'.")
+            else:
+                msg_text = ("No numerical integration method specified, "
+                            "choosing method '{method}'.")
+            logger.info(msg_text.format(group_name=group_name,
+                                        method=the_method), 'method_choice')
```

Comment:
What about two messages as I proposed above? On the one hand, it would be nice to give an indication about what takes so much time in the preparation phase, on the other hand it will be a bit verbose for those cases where it is quick. It will also be obsolete as soon as we introduce a time-dependent warning, but this not something we'll implement in the short term.